scripts/help-text/release.txt

```diff
@@ -1,9 +1,13 @@
 Usage: ./scripts/release [OPTION]... [TARGET_COMMIT]
-Releases a new version of Rest.li by creating and pushing a tag at TARGET_COMMIT (defaults to HEAD).
-This script must be run from the root project directory.
+Releases a new version of Rest.li by creating and pushing a tag at TARGET_COMMIT (defaults to HEAD). This script must be
+run from the root project directory. TARGET_COMMIT must be an ancestor of master, unless the version being released is a
+release candidate version.
```

Comment:
Does it mean if TARGET is at non-master branch, then it can be released as a rc?

Reply:
It means:
- Normal releases must be an ancestor of master.
- Candidate releases can be on any branch.